zopkio/__init__.py

```diff
@@ -19,4 +19,4 @@
 
 """Zopkio test framework"""
 
-__version__ = '0.1.19'
+__version__ = '0.2.0'
```

Comment:
"No newline at end of file" - Will this trigger some precommit error if we enable flake8 rule check etc.?